interpreter/interpreter.py

```diff
@@ -315,15 +352,28 @@ def chat(self, message=None, return_messages=False):
     # ^ verify_api_key may set self.local to True, so we run this as an 'if', not 'elif':
     if self.local:
 
+      # Load the model configuration
```

Comment:
should this code block (line 356-364) moves below line 366, which will only trigger when self.llama_instance == None:

Reply:
Okay i updated this logic to load if llama model is None.
It tries to load from configuration first if model was not found then it will try to install it.